pkg/sfu/downtrack.go

```diff
@@ -434,6 +435,9 @@ func (d *DownTrack) WriteRTP(extPkt *buffer.ExtPacket, layer int32) error {
 		if tp.isDroppingRelevant {
 			d.pktsDropped.Inc()
 		}
+		if err != nil {
+			d.logger.Errorw("write rtp packet failed", err)
```

Comment:
is it intentional to use same log message for all these log?

Reply:
Yeah, the `err` would show the underlying reason.